### PR TITLE
Improve debugging

### PR DESCRIPTION
**Pull request by harbu**
I required some more detailed debugging to understand why my integration test is not working.

Some Changes
- Slightly re-worded some debug messages and added in values of variables
- Logic layer debug messages: added identity of peer to debug messages, so we see who is reporting. This is important for tests that are running multiple nodes for example. 
- Use the 4 last characters of PeerInfo id for peer identity.

Example of how logic layer logging looks like now:
```
  streamr:logic:tracker:5AuC started 5AuC +0ms
  streamr:logic:node:joEY started joEY +0ms
  streamr:logic:node:8V2S started 8V2S +0ms
  streamr:logic:node:46N4 started 46N4 +0ms
  streamr:logic:node:WjkE started WjkE +0ms
  streamr:logic:node:joEY connected to tracker 5AuC +5s
  streamr:logic:node:joEY sending status to tracker 5AuC +0ms
  streamr:logic:node:joEY requesting more peers from tracker 5AuC +0ms
  streamr:logic:node:8V2S connected to tracker 5AuC +5s
  streamr:logic:node:8V2S sending status to tracker 5AuC +0ms
  streamr:logic:node:8V2S requesting more peers from tracker 5AuC +0ms
  streamr:logic:node:46N4 connected to tracker 5AuC +5s
  streamr:logic:node:46N4 sending status to tracker 5AuC +0ms
  streamr:logic:node:46N4 requesting more peers from tracker 5AuC +0ms
  streamr:logic:node:WjkE connected to tracker 5AuC +5s
  streamr:logic:node:WjkE sending status to tracker 5AuC +0ms
  streamr:logic:node:WjkE requesting more peers from tracker 5AuC +0ms
  streamr:logic:tracker:5AuC no available nodes to send to joEY +12s
  streamr:logic:tracker:5AuC no available nodes to send to 8V2S +4ms
  streamr:logic:tracker:5AuC no available nodes to send to 46N4 +1ms
  streamr:logic:tracker:5AuC no available nodes to send to WjkE +0ms
  streamr:logic:tracker:5AuC received from joEY status {"streams":[],"started":"2018-9-19 11:10:41"} +69ms
  streamr:logic:tracker:5AuC received from 8V2S status {"streams":[],"started":"2018-9-19 11:10:41"} +3ms
  streamr:logic:tracker:5AuC received from 46N4 status {"streams":[],"started":"2018-9-19 11:10:41"} +1ms
  streamr:logic:tracker:5AuC received from WjkE status {"streams":[],"started":"2018-9-19 11:10:41"} +0ms
  streamr:logic:tracker:5AuC sending list of 3 nodes to joEY +15ms
  streamr:logic:tracker:5AuC sending list of 3 nodes to 8V2S +2ms
  streamr:logic:tracker:5AuC sending list of 3 nodes to 46N4 +0ms
  streamr:logic:tracker:5AuC sending list of 3 nodes to WjkE +0ms
  streamr:logic:node:joEY ask tracker 5AuC who is responsible for stream stream-1 +219ms
  streamr:logic:node:WjkE ask tracker 5AuC who is responsible for stream stream-2 +213ms
  streamr:logic:node:joEY ask tracker 5AuC who is responsible for stream stream-1 +1ms
  streamr:logic:node:WjkE ask tracker 5AuC who is responsible for stream stream-2 +1ms
  streamr:logic:node:joEY ask tracker 5AuC who is responsible for stream stream-1 +0ms
  streamr:logic:node:WjkE ask tracker 5AuC who is responsible for stream stream-2 +0ms
  streamr:logic:node:joEY ask tracker 5AuC who is responsible for stream stream-1 +0ms
  streamr:logic:node:WjkE ask tracker 5AuC who is responsible for stream stream-2 +0ms
  streamr:logic:node:joEY ask tracker 5AuC who is responsible for stream stream-1 +1ms
  streamr:logic:node:WjkE ask tracker 5AuC who is responsible for stream stream-2 +1ms
  streamr:logic:tracker:5AuC looking for stream stream-1 +529ms
  streamr:logic:tracker:5AuC stream stream-1 assigned to joEY +0ms
  streamr:logic:tracker:5AuC looking for stream stream-2 +0ms
  streamr:logic:tracker:5AuC stream stream-2 assigned to WjkE +0ms
  streamr:logic:tracker:5AuC looking for stream stream-1 +10ms
  streamr:logic:tracker:5AuC stream stream-1 assigned to joEY +0ms
  streamr:logic:tracker:5AuC looking for stream stream-2 +0ms
  streamr:logic:tracker:5AuC stream stream-2 assigned to WjkE +0ms
  streamr:logic:tracker:5AuC looking for stream stream-1 +11ms
  streamr:logic:tracker:5AuC stream stream-1 assigned to joEY +0ms
  streamr:logic:tracker:5AuC looking for stream stream-2 +0ms
  streamr:logic:tracker:5AuC stream stream-2 assigned to WjkE +0ms
  streamr:logic:tracker:5AuC looking for stream stream-1 +15ms
  streamr:logic:tracker:5AuC stream stream-1 assigned to joEY +0ms
  streamr:logic:tracker:5AuC looking for stream stream-2 +0ms
  streamr:logic:tracker:5AuC stream stream-2 assigned to WjkE +1ms
  streamr:logic:tracker:5AuC looking for stream stream-1 +10ms
  streamr:logic:tracker:5AuC stream stream-1 assigned to joEY +0ms
  streamr:logic:tracker:5AuC looking for stream stream-2 +0ms
  streamr:logic:tracker:5AuC stream stream-2 assigned to WjkE +0ms
  streamr:logic:node:joEY stream stream-1 added to own streams +565ms
  streamr:logic:node:joEY sending status to tracker 5AuC +0ms
  streamr:logic:node:WjkE stream stream-2 added to own streams +566ms
  streamr:logic:node:WjkE sending status to tracker 5AuC +0ms
  streamr:logic:node:joEY stream stream-1 added to own streams +10ms
  streamr:logic:node:joEY sending status to tracker 5AuC +0ms
  streamr:logic:node:WjkE stream stream-2 added to own streams +9ms
  streamr:logic:node:WjkE sending status to tracker 5AuC +0ms
  streamr:logic:node:joEY stream stream-1 added to own streams +10ms
  streamr:logic:node:joEY sending status to tracker 5AuC +0ms
  streamr:logic:node:WjkE stream stream-2 added to own streams +10ms
  streamr:logic:node:WjkE sending status to tracker 5AuC +0ms
  streamr:logic:node:joEY stream stream-1 added to own streams +7ms
  streamr:logic:node:joEY sending status to tracker 5AuC +0ms
  streamr:logic:node:WjkE stream stream-2 added to own streams +7ms
  streamr:logic:node:WjkE sending status to tracker 5AuC +0ms
  streamr:logic:node:joEY stream stream-1 added to own streams +16ms
  streamr:logic:node:joEY sending status to tracker 5AuC +0ms
  streamr:logic:node:WjkE stream stream-2 added to own streams +16ms
  streamr:logic:node:WjkE sending status to tracker 5AuC +0ms
  streamr:logic:tracker:5AuC received from joEY status {"streams":["stream-1"],"started":"2018-9-19 11:10:41"} +211ms
  streamr:logic:tracker:5AuC received from WjkE status {"streams":["stream-2"],"started":"2018-9-19 11:10:41"} +0ms
  streamr:logic:tracker:5AuC received from joEY status {"streams":["stream-1"],"started":"2018-9-19 11:10:41"} +8ms
  streamr:logic:tracker:5AuC received from WjkE status {"streams":["stream-2"],"started":"2018-9-19 11:10:41"} +0ms
  streamr:logic:tracker:5AuC received from joEY status {"streams":["stream-1"],"started":"2018-9-19 11:10:41"} +12ms
  streamr:logic:tracker:5AuC received from WjkE status {"streams":["stream-2"],"started":"2018-9-19 11:10:41"} +0ms
  streamr:logic:tracker:5AuC received from joEY status {"streams":["stream-1"],"started":"2018-9-19 11:10:41"} +8ms
  streamr:logic:tracker:5AuC received from WjkE status {"streams":["stream-2"],"started":"2018-9-19 11:10:41"} +0ms
  streamr:logic:tracker:5AuC received from joEY status {"streams":["stream-1"],"started":"2018-9-19 11:10:41"} +9ms
  streamr:logic:tracker:5AuC received from WjkE status {"streams":["stream-2"],"started":"2018-9-19 11:10:41"} +1ms
  streamr:logic:tracker:5AuC stopping +215ms
```

### CODE REVIEW

**Review by ptesavol — APPROVED** [review]
Looks ok.

File: src/logic/Node.js [hunk 1]
```diff
@@ -16,11 +16,12 @@ class Node extends EventEmitter {
         this.knownStreams = new Map()
         this.ownStreams = new Set()
 
-        this.id = generateClientId('node')
+        this.id = getIdShort(nodeToNode.connection.node.peerInfo) // TODO: better way?
```
Comment (mirpo) [inline src/logic/Node.js:19]: lol

File: src/logic/Node.js [hunk 2]
```diff
@@ -80,6 +84,7 @@ class Node extends EventEmitter {
     }
 
     stop(cb) {
+        this.debug('stopping')
         this.protocols.trackerNode.stop(cb)
```
Comment (mirpo) [inline src/logic/Node.js:88]: we are using one libp2p instance, but you're stopping both

**Review by mirpo — APPROVED** [review]
a lot of debug now 🥇

**Pull request merged**